dotnet/src/InternalUtilities/planning/PlannerInstrumentation.cs

```diff
@@ -12,17 +12,26 @@ namespace Microsoft.SemanticKernel.Planning;
 /// <summary>Surrounds the invocation of a planner with logging and metrics.</summary>
 internal static partial class PlannerInstrumentation
 {
-    /// <summary><see cref="ActivitySource"/> for planning-related activities.</summary>
-    private static readonly ActivitySource s_activitySource = new("Microsoft.SemanticKernel.Planning");
-
-    /// <summary><see cref="Meter"/> for planner-related metrics.</summary>
-    private static readonly Meter s_meter = new("Microsoft.SemanticKernel.Planning");
-
     /// <summary><see cref="Histogram{T}"/> to record plan creation duration.</summary>
-    private static readonly Histogram<double> s_createPlanDuration = s_meter.CreateHistogram<double>(
-        name: "sk.planning.create_plan.duration",
-        unit: "s",
-        description: "Duration time of plan creation.");
+    private static readonly Histogram<double> s_createPlanDuration =
+        PlanningInstrumentation.Meter.CreateHistogram<double>(
+            name: "sk.planning.create_plan.duration",
+            unit: "s",
+            description: "Duration time of plan creation.");
+
+    /// <summary><see cref="Counter{T}"/> to record planner invocation success counts.</summary>
+    private static readonly Counter<int> s_invocationSuccess =
+        PlanningInstrumentation.Meter.CreateCounter<int>(
+            name: "sk.planning.create_plan.success",
+            unit: "{invocation}",
+            description: "Measures the number of successful planner executions");
+
+    /// <summary><see cref="Counter{T}"/> to record planner invocation failure counts.</summary>
+    private static readonly Counter<int> s_invocationFailure =
+        PlanningInstrumentation.Meter.CreateCounter<int>(
+            name: "sk.planning.create_plan.failure",
+            unit: "{invocation}",
+            description: "Measures the number of failed planner executions");
```

Comment:
Same comment: we shouldn't need these. All of the relevant information should be available via the single s_createPlanDuration: if a recording is tagged as error.type, then it was a failure, otherwise it was a success.